### PR TITLE
modify example in readme to make an example work for nodejs instead f…

### PR DESCRIPTION
**Pull request by kafeltz**
…or browser-only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,17 @@ console.log(profile.pubkey)
 // prints: 32e1827635450ebb3c5a7d12c1f8e7b2b514439ac10a67eef3d9fd9c5c68e245
 console.log(profile.relays)
 // prints: [wss://relay.damus.io]
+
+// or alternative for nodejs
+import 'websocket-polyfill'
+import fetch from 'node-fetch';
```
Comment (ralyodio) [inline README.md:169]: this isn't required anymore with node.

File: README.md [hunk 2]
```diff
@@ -163,6 +163,17 @@ console.log(profile.pubkey)
 // prints: 32e1827635450ebb3c5a7d12c1f8e7b2b514439ac10a67eef3d9fd9c5c68e245
 console.log(profile.relays)
 // prints: [wss://relay.damus.io]
+
+// or alternative for nodejs
+import 'websocket-polyfill'
```
Comment (ralyodio) [inline README.md:168]: what browsers don't support websockets?

Comment (kafeltz) [reply README.md:168]: It's for nodejs. Besides, it's recommended to do it in the readme.md on the main page.

File: README.md (hunk 1 above)
Comment (kafeltz) [reply README.md:169]: Yes, It is. 
I'm using v16.20.0.

Comment (ralyodio) [reply README.md:169]: latest lts is 20

Comment (kafeltz) [reply README.md:169]: The world is not only black and white.

**Comment by fiatjaf** [comment]
Is this solved? Sorry, I forgot about it.

**Pull request closed without merging**